src/types/index.d.ts

```diff
@@ -4,3 +4,11 @@ declare module "jspdf" {
     getPageHeight: () => number;
   }
 }
+
+declare module "@mapbox/mapbox-gl-rtl-text" {
```

Comment:
why do we need to declare this?
doesn't the module have its own TS declaration file?

Reply:
mapbox is still posessed by Flow, they have no declaration file